delta-api/src/main/java/io/cdap/delta/api/DMLEvent.java

```diff
@@ -193,9 +198,9 @@ public DMLEvent build() {
       int sizeInBytes = (operationType == DMLOperation.Type.INSERT ||
         operationType == DMLOperation.Type.UPDATE) ? computeSizeInBytes(row) : 0;
 
-      return new DMLEvent(offset, database, new DMLOperation(table, operationType, ingestTimestampMillis, sizeInBytes),
-                          row, previousRow, transactionId, ingestTimestampMillis, sourceTimestampMillis, isSnapshot,
-                          rowId);
+      return new DMLEvent(offset, databaseName,
```

Comment:
We can remove databaseName from DMLEvent since its already part of DMLOperation.

Reply:
done :)